### ISSUE TITLE
Egg incubator broke after a while

### ISSUE THREAD

**Issue by Zth--** [vgstation-coders/vgstation13#6163]
I was playing on deff, ordered the crate and builded it in cargo. So I used it two times and it worked nice. But then, the third time the machine would do nothing, the counter stays at zero. So I try rebuilding it, changing its components and moving it, but nothing. 
There was ANOTHER egg incubator that worked just fine at hydroponics (I tried with the same egg) but mine was broken

**Comment by Shadowmech88** [comment]
I know there's a cap on the number of animal children that can exist in the world at one time.
Is it possible that you reached the cap, which made the machine stop working, and one or more chickens died between using that machine and using the other machine?
How many chickens did you have?

**Comment by Zth--** [comment]
'''if(animal_count[/mob/living/simple_animal/chicken] >= ANIMAL_CHILD_CAP)
user << "<span class='warning'>You get the feeling there are enough chickens already.</span>"'''

Also, I repeat, one egg incubator was working while the other one not

**Comment by Kurfursten** [comment]
Can you tell us more? Did you eject the egg and try a different egg? Did you have someone VV on it to see if it had weird contents or something?

**Comment by Zth--** [comment]
I tried with different eggs, yes. Also I told admins three times but well, they didn't listen. I even tried moving the machine to other area, but nothing.

**Comment by Kurfursten** [comment]
After ~20 shifts of testing incubators (and way too much botanist), I reproduced this successfully only once. Probe was on at the time and we couldn't find anything in the VV. It is exactly like he said, though: disassembling and reassembling didn't fix it. Very, very confusing.

It's only later, after the shift, that I consider: what if something was wrong with the egg itself? That is how I could best explain the problem persisting through reconstruction.

**Comment by Zth--** [comment]
Like I said before, I tested with different eggs without any success. I don't know what the fuck goes on here

**Comment by Zth--** [comment]
What if something is wrong with the code that crestes this machine?

**Comment by Kurfursten** [comment]
It uses the same constructable frame code as everything else.

Until we can reproduce it, I'm not sure what to say, but I haven't personally seen this bug again whether with the real egg incubator OR the box cloner.

**Comment by Zth--** [comment]
no longer relevant
